tests/test_flask.py

```diff
@@ -432,6 +433,27 @@ def test_ical_meeting(self):
         )
         self.assertEqual(data, expected_data)
 
+        # Test meeting iCal with reminder
+        expected_data = self.get_sample_file_content(
+            'meeting_with_reminder.ical').format(
+            start_datetime=datetime.combine(
+                meeting_obj.meeting_date,
+                meeting_obj.meeting_time_start).strftime(
+                    '%Y%m%dT%H%M%SZ'),
+            end_datetime=datetime.combine(
+                meeting_obj.meeting_date_end,
+                meeting_obj.meeting_time_stop).strftime(
+                    '%Y%m%dT%H%M%SZ')
+        )
+        output = self.app.get('/ical/calendar/meeting/2/?reminder_delta=60')
```

Comment:
Could we test with a reminder_delta that is not an int? Also test with a reminder_delta that is not in the default choice (ie: 90)?